airqo-platform/AirQo-api
# [hotfix] JSON response for recent feeds

- **What does this PR do?**

This PR changes the response for recent data from feeds. 
It also addresses the issue of incorrect GPS coordinates. 

- **What JIRA task is related to this PR?**

No link available

- **How do I test out this PR?**

 `cd AirQo-api/src/data-mgt`
Windows: `npm run dev-pc` or `npm run stage-pc`
Mac:  `npm run dev-mac` or` npm run stage-mac`

- **Which end points should be tested?**
**Get most recent device data:** http://localhost:3000/api/v1/data/feeds/recent/{CHANNEL ID}

RESPONSE:
![image](https://user-images.githubusercontent.com/1590213/90155999-a4eb3e00-dd94-11ea-90d5-6b49fa3507e9.png)

More details about this ready endpoint can be found [here](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=897606332)
